workflow-templates/im-deploy-az-database.yml

```diff
@@ -28,18 +28,18 @@ on:
   workflow_dispatch:
     inputs:
       tag:
-        description: The tag for the release that will be deployed.  For Production, only tags reachable by the default branch will be accepted.
-        required: true
-      environment:
-        description: The environment to deploy to
+        description: The tag for the release that will be deployed.  For Production, only tags reachable by the default branch will be accepted. Leave blank to use the workflow's branch.
+        required: false
```

Comment:
I think the `tag` input should continue to be required like in all the other deployment workflows, and the Verify Tag Exists step should still be used early in the workflow in the `set-vars` job.

Reply:
I do this in all my repos.  I use the workflow tag except for prod.  You have to type it in.  This lets people use the github ui to select the tag for all lower environments.